light_node/src/main.rs

```diff
@@ -252,6 +253,12 @@ fn main() {
     // Creates default logger
     let log = create_logger(&env);
 
+    // Need unsafe to access direct libc syscall interface
+    unsafe {
```

Comment:
@michal-stanek 
what if we put unsafe {} inside init_limits?  and change `pub unsafe fn init_limits` -> `pub fn init_limits`?
maybe we could put there more functionalities in future, which can be also "safe",
something like this: https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#creating-a-safe-abstraction-over-unsafe-code

Reply:
Good idea, done.